src/index.js

```diff
@@ -1,3 +1,20 @@
+// handles keypresses for accessibility
+
+function handleClick(e){
+    if (e.type === 'keypress'){
+        let code = e.keyCode;
```

Comment:
I think we can use `const` here instead:
```suggestion
        const code = e.keyCode;
```